data/com.github.melix99.telegrand.metainfo.xml.in.in

```diff
@@ -27,6 +27,13 @@
   <url type="bugtracker">https://github.com/melix99/telegrand/issues</url>
   <url type="donation">https://github.com/sponsors/melix99</url>
   <content_rating type="oars-1.0" />
+  <custom>
+    <value key="Purism::form_factor">workstation</value>
+    <value key="Purism::form_factor">mobile</value>
+  </custom>
```

Comment:
Mh, I think these aren't needed anymore these days? Why do we need them?

Reply:
Not sure, telegrand is currently displayed as desktop only in gnome software, but maybe it's a bug of the ci or maybe it's because not being added to the flathub/any other remote.
![Screenshot from 2023-03-28 00-10-18](https://user-images.githubusercontent.com/110191114/228079288-87a4c963-0630-4280-97ee-2fd6e3f82332.png)

Reply:
I should investigate it before commiting this